### ISSUE TITLE
Make existential deposit limits more user-friendly

### ISSUE THREAD

**Issue by JustLuuuu** [kodadot/nft-gallery#10198]
Issue
---

- It's not the first time this has happened, especially with new users that are coming to Polkadot. New users usually come to platform with 1-3 DOTs to test and play with it. And they cannot buy NFTs. 
- existential deposits are unclear right now - because of the issue with the relay chain, they were raised on Koda (1,5 on DOt, 0.5 on AHP) - will we have that forever? I can add this to docs but this doesn't help at all as people are not going to search docs for answers. 

<img width="733" alt="Screenshot 2024-05-03 at 10 27 41" src="https://github.com/kodadot/ops-internal/assets/104380776/c05646b4-0f59-4a21-afd1-ea49b03996cb">

<img width="734" alt="Screenshot 2024-05-03 at 10 27 30" src="https://github.com/kodadot/ops-internal/assets/104380776/c69c7168-e670-4f4e-abf9-f772747b40d5">

Solution
---
- we need to make existential deposit limits **more clear** 
- How I imagine it will work: you should **not be able** to click on "buy" or proceed if you wanna buy NFTs if it makes you go under ED - at the moment looks like the transaction can happen but it will fail (frustrating for users). Instead of that there should be a message informing them: You cannot buy this NFT as 1,5 DOT must stay in your account as an existential deposit - and ED linked to hello page - https://hello.kodadot.xyz/multi-chain/existential-deposit).
- anyone trying to buy NFT with ED should know why they cannot. 
- this should be clear with all transactions - not only buying, but it needs to be clear with all transactions - burning, listing, selling... 
- @prury can you test how it is right now? how app reacts when people are trying to buy NFTs with ED?
- @kodadot/internal-dev - as this affects most of the new users I added p1. 

Problem two
---
- till when we will have raised deposits on Koda? Is it really a necessity?

**Comment by hassnian** [comment]
> we need to make existential deposit limits more clear
>anyone trying to buy NFT with ED should know why they cannot.
this should be clear with all transactions - not only buying, but it needs to be clear with all transactions - burning, listing, selling...

cc @exezbcz 

> will we have that forever?

as mentioned [here](https://github.com/kodadot/nft-gallery/issues/9596#issuecomment-1994597629) at this moment this is the only solution.

>  New users usually come to platform with 1-3 DOTs to test and play with it. And they cannot buy NFTs.

regarding EDs `1.5 DOT` on polkadot and `0.5 DOT` on ahp  are **only applied when doing teleports**

if you have funds already in ahp the current ED is set to `0.01 DOT`  and polkadot `1 DOT`, and these EDs are used when performing actions like minting, buying etc

 > at the moment looks like the transaction can happen but it will fail 

I'm assuming the user had around `2.4 DOT` , and was able to do one teleport and mint, the second time with 1.9 DOT. would go below our ED. worked as intended , so he wasn't able to sign that 's why he saw the `add funds`

@prury can you test this , users should not be able to sign the transaction if it goes below ED

**Comment by JustLuuuu** [comment]
> I'm assuming the user had around `2.4 DOT` , and was able to do one teleport and mint, the second time with 1.9 DOT. would go below our ED. worked as intended , so he wasn't able to sign that 's why he saw the `add funds`

Yeah and here is the problem. People coming from outside of Polkadot don't have any idea what ED is. They assume that its not working and there is an issue. In this case they should be able to see a message or info somewhere. For example, Your funds cannot go under existential deposit.  Or something like that linked to hello. 😄 

I mean, these are gen artists that are currently struggling; I believe if artists are struggling, users must be double confused.

**Comment by prury** [comment]
>
>     * @prury can you test how it is right now? how app reacts when people are trying to buy NFTs with ED?

it only shows add funds via onramp, without any info about ED. 

> 
> @prury can you test this , users should not be able to sign the transaction if it goes below ED, thanks

tested and they are not able to go trough if transaction goes below ED

> Yeah and here is the problem. People coming from outside of Polkadot don't have any idea what ED is. They assume that its not working and there is an issue. In this case they should be able to see a message or info somewhere. For example, Your funds cannot go under existential deposit. Or something like that linked to hello. 😄

on point Lu, transfer already has some information about ED, we need to put it on drops as well. 

summoning @exezbcz to provide a design for it, be it a tooltip or a small text to be shown together with the add funds via on ramp modal and also inside the auto teleport modal.
